uk.co.compucorp.civicrm.tasksassignments/js/test/tasks-assignments/controllers/modal/modal-assignment.controller.spec.js

```diff
@@ -114,7 +136,10 @@ define([
         });
 
         scope.taskList.forEach(function (task) {
-          _.assign(task, { activity_date_time: '2013-01-01', assignee_contact_id: [3] });
+          _.assign(task, {
+            assignee_contact_id: [ expectedContactId ],
+            activity_date_time: '2013-01-01'
```

Comment:
Could we please move '2013-01-01' to a variable somewhere else so we don't have that constant. Something like:

`var sampleDate = '2013-01-01';`

In that case I understand this is just a sample date and it can be essentially any date and there is nothing special about 2013 New Year.